components/tikv_util/src/sys/mod.rs

```diff
@@ -156,3 +160,41 @@ pub fn cache_size(level: usize) -> Option<u64> {
 pub fn cache_line_size(level: usize) -> Option<u64> {
     read_size_in_cache(level, "coherency_line_size")
 }
+
+pub fn path_in_same_mount_point(path1: &str, path2: &str) -> bool {
+    if path1.is_empty() || path2.is_empty() {
+        return true;
+    }
+    let path1 = PathBuf::from(path1);
+    let path1_mount_point = match get_mount(&path1) {
```

Comment:
A more concise code style is recommended:

```Rust
let (path1, path2) = (PathBuf::from(path1), PathBuf::from(path2));
match (get_mount(&path1), get_mount(&path2)) {
    (Ok(Some(a)), Ok(Some(b))) => {
        return a == b;
    }
    (Ok(None), Ok(_)) => {
         warn!(...);
        return false;
    }
    (Ok(_), Ok(None)) => {
         warn!(...);
        return false;
    }
    _ => {
        error(....);
        return false;
    }
}

```

Reply:
Good Suggestion, thanks!